directlfq/utils.py

```diff
@@ -256,8 +256,10 @@ def add_columns_to_lfq_results_table(lfq_results_df, input_file, columns_to_add)
     all_columns = columns_to_add + [protein_column_input_table] + standard_columns_for_input_type
     all_columns = filter_columns_to_existing_columns(all_columns, input_file)
 
-    input_df = pd.read_csv(input_file, sep="\t", usecols=all_columns).drop_duplicates(subset=protein_column_input_table)
     lfq_results_df = lfq_results_df[[x is not None for x in lfq_results_df[config.PROTEIN_ID]]]
+    if len(all_columns) == 1: #if there are no columns to add, return the original dataframe
```

Comment:
I see some discrepancy between the comment and the logic above:
`columns_to_add + [protein_column_input_table] + standard_columns_for_input_type`

Reply:
so both the columns to add (columns specified by the user) and the standard columns (columns added by default) can be empty lists. I now made the if statement more explicit.

`    if (len(columns_to_add) == 0) and (len(standard_columns_for_input_type)==0) : #if there are no columns to add and no additional standard columns defined return the original dataframe
        return lfq_results_df `

Reply:
sounds good, I think you could even drop the comment then as the code reads almost as good as plain text ;-)

Reply:
yess ☺️